docs/Actions/apply-tokens.md

```diff
@@ -0,0 +1,165 @@
+
+
+```json
+
+{
+    "Title": "Apply Tokens",
+    "ActionType": "ApplyTokens",
+    "Parameters": {
+        "Tokens": [
+            {
+                "value": "[DoMath:Eval(expression=[User:UserId]*2)]",
+                "name": "DoubleOfUserId"
+            },
+            {
+                "value": "[DoMath:Eval(expression=(5*2)+2)]",
+                "name": "DoubleOfFivePlusTwo"
+            }
+        ],
+        "RecursionCount": "1"
+    }
+}
+​
+```
+
+### `2. Apply tokens inside a template (Recursion >1)`
+
+This example will show you how to replace tokens inside another token.
+
+```json
+
+{
+    "Title": "Execute Actions",
+    "ActionType": "ExecuteActions",
+    "Description": "Apply tokens inside a template",
+    "Parameters": {
+        "ActionList": [
+            {
+                "Title": "Apply Tokens",
+                "ActionType": "ApplyTokens",
+                "Parameters": {
+                    "Tokens": [
+                        {
+                            "value": "Hi, [User:FirstName] [User:LastName]. Please contact me!",
+                            "name": "EmailTemplate"
+                        }
+                    ],
+                    "RecursionCount": "1"
+                },
+                "$_uid": "action15910976849431312",
+                "$_isOpen": true,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Description": "Assign Template to Token"
+            },
+            {
+                "Title": "Apply Tokens",
+                "ActionType": "ApplyTokens",
+                "Parameters": {
+                    "Tokens": [
+                        {
+                            "value": "[EmailTemplate]",
+                            "name": "FinalTemplate"
+                        }
+                    ],
+                    "RecursionCount": "2"
+                },
+                "$_uid": "action1591097684943656",
+                "$_isOpen": true,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Id": -1,
+                "Description": "Apply tokens twice"
+            }
+        ]
+    }
+}
+
+```
+
+## `Frequently asked questions`
+
+**What is the best Number of Recursion**
+
+Usually 2 or 3 depending of the situation should be enough but it really depends on what you need to achieve. Setting the number of recursion bigger that is need it will not have any impact on the result but will impact the performance. If you are not sure how recursion works increment the value by one until you get the desire output and there are no tokens left. That will be the desire value for the execution.
+
+**How long can I use the output token?**
+
+It can be used after the action is executed in the execution flow and/or Workflow and it be available until the executions ends or it is stopped. 

```

Comment:
Avoid using execute 3 times in a sentence. Split in multiple sentences or find equivalent expressions.